packages/suite/src/utils/suite/homescreen.ts

```diff
@@ -228,8 +228,5 @@ export const imagePathToHex = async (imagePath: string, deviceModel: DeviceModel
 export const isHomescreenSupportedOnDevice = (device: TrezorDevice) => {
     const deviceModel = getDeviceModel(device);
 
-    return (
-        deviceModel !== DeviceModel.TT ||
-        (deviceModel === DeviceModel.TT && device.features?.homescreen_format === 'Jpeg240x240')
-    );
+    return deviceModel !== DeviceModel.TT || device.features?.homescreen_format !== undefined;
```

Comment:
```suggestion
   return (
        deviceModel !== DeviceModel.TT ||
        (deviceModel === DeviceModel.TT && device.features?.homescreen_format)
    );
```

Homescreen on T1B1 and T2B1 is supported all the time.
On T2T1 it is supported only from the time `homescreen_format` is defined (`v2.6.0`)